common/src/main/java/net/william278/huskhomes/config/Settings.java

```diff
@@ -340,6 +340,10 @@ public boolean isWorldRtpRestricted(@NotNull World world) {
                     .map(n -> n.startsWith("minecraft:") ? n.substring(10) : n)
                     .anyMatch(n -> n.equalsIgnoreCase(filteredName));
         }
+
+        @Comment({"List of server in which /rtp is allowed. (Only relevant when using cross server mode WITH REDIS)",
+                "If a server is not defined here the RTP logic has no way of knowing its existence."})
+        private List<String> allowedServers = List.of("server-01", "server-02");
```

Comment:
`randomTargetServers` instead of "allowed"? 

I'd also like a big n' simple `true`/`false` toggle for cross-server RTP ideally